File: beautiful_days/bd.py
def beautiful_days(i:int, j:int, k:int) -> int:
    count = 0
    for x in range(i, j+1):
        if abs(int(str(x)[::-1])-x) % k == 0:
            count += 1
    return count

if __name__ == "__main__":
    print(beautiful_days(20, 23, 6))

File: beautiful_days/bd.js
const beautifulDays = (i, j, k) => {
    let count = 0;
    for(let x = i; x<=j; x++){
        let reverse = parseInt(x.toString().split('').reverse().join(''));
        if(Math.abs(x-reverse)%k === 0){
            count++;
        }
    }
    return count;
}

console.log(beautifulDays(20,23,6));